### DISCUSSION TITLE
Raspberry Pi 4/5 mit ubuntu64 und push_firmware funktioniert nicht.

### DISCUSSION THREAD

**Discussion by FireDVB** [Freetz-NG/freetz-ng#958] [push firmware]
Wenn ich das im Terminal eingebe:

sudo ./push_firmware 7530AX_07.80.image -mf -cmd ftp -afu

kommt folgene Meldung:

File ./fit/fdtdump missing. Run 'make dtc-host' first.

Kann da wer helfen?

**Comment by Conan179** [comment]
"Run 'make dtc-host' first."?

**Reply by FireDVB** [reply]
make dtc-host

make: *** Keine Regel, um „dtc-host“ zu erstellen.  Schluss.

**Reply by Conan179** [reply]
ohne sudo
  cd ~/freetz-ng
  make dtc-host

**Reply by FireDVB** [reply]
Dann kommt das:

make dtc-host
Freetz-NG 23285-c7a4fc8f5 master 2024-03-09
ERROR: The program 'libtool' was not found in path
ERROR: The program 'svn' was not found in path
ERROR: The header file 'sys/acl.h' was not found in /usr/(local/)include
ERROR: The header file 'sys/capability.h' was not found in /usr/(local/)include
ERROR: The header file 'uuid/uuid.h' was not found in /usr/(local/)include
ERROR: The header file 'openssl/evp.h' was not found in /usr/(local/)include
ERROR: The header file 'gnutls/gnutls.h' was not found in /usr/(local/)include
ERROR: The header file 'sqlite3.h' was not found in /usr/(local/)include
ERROR: The pkgconf 'libzstd.pc' was not found in /usr/(local/)lib*/(*/)pkgconfig/
Makefile:191: *** Some prerequisites are missing! Install the missing packages with 'tools/prerequisites install' or check https://freetz-ng.github.io/freetz-ng/PREREQUISITES for hints. See '.prerequisites' for why.  Schluss.
make: *** [Makefile:46: envira] Beendet

**Reply by Conan179** [reply]
"Makefile:191: *** Some prerequisites are missing! Install the missing packages with 'tools/prerequisites install' or check https://freetz-ng.github.io/freetz-ng/PREREQUISITES for hints. See '.prerequisites' for why. Schluss."

**Reply by FireDVB** [reply]
Habe ich alles gemacht aber manche werden einfach nicht installiert:

sudo apt-get -y install   autopoint bc binutils bison bsdmainutils bzip2 ccache cmake cpio curl ecj flex ftp g++ gawk gcc   gcc-multilib gettext git graphicsmagick imagemagick inkscape intltool java-wrappers kmod   lib32ncurses-dev lib32stdc++6 lib32z1-dev libacl1-dev libc6-dev-i386 libcap-dev libelf-dev   libglib2.0-dev libgnutls28-dev libncurses-dev libreadline-dev libsqlite3-dev libssl-dev   libstring-crc32-perl libtool-bin libusb-dev libxml2-dev libzstd-dev make ncftp netcat-openbsd net-tools patch   patchutils perl pkg-config pv rsync sharutils sqlite3 subversion sudo texinfo tofrodos unar unzip   uuid-dev wget zip zlib1g-dev

**Reply by Conan179** [reply]
""Makefile:191: *** Some prerequisites are missing! Install the missing packages with 'tools/prerequisites install'

**Reply by FireDVB** [reply]
./prerequisites install -y Ubuntu23

Packages for 'Ubuntu23':
autopoint bc binutils bison bsdmainutils bzip2 ccache cmake cpio curl ecj flex ftp g++ gawk gcc gcc-multilib gettext git graphicsmagick imagemagick inkscape intltool java-wrappers kmod lib32ncurses-dev lib32stdc++6 lib32z1-dev libacl1-dev libc6-dev-i386 libcap-dev libelf-dev libglib2.0-dev libgnutls28-dev libncurses-dev libreadline-dev libsqlite3-dev libssl-dev libstring-crc32-perl libtool-bin libusb-dev libxml2-dev libzstd-dev make ncftp netcat-openbsd net-tools patch patchutils perl pkg-config pv rsync sharutils sqlite3 subversion sudo texinfo tofrodos unar unzip uuid-dev wget zip zlib1g-dev

OK:1 http://ports.ubuntu.com/ubuntu-ports mantic InRelease
OK:2 http://ports.ubuntu.com/ubuntu-ports mantic-updates InRelease
OK:3 http://ports.ubuntu.com/ubuntu-ports mantic-security InRelease
Ign:4 http://ports.ubuntu.com/ubuntu-ports mantic/main i386 Packages
Holen:5 http://ports.ubuntu.com/ubuntu-ports mantic/main Translation-de [508 kB]
Holen:6 http://ports.ubuntu.com/ubuntu-ports mantic/restricted i386 Packages [32,3 kB]
Holen:6 http://ports.ubuntu.com/ubuntu-ports mantic/restricted i386 Packages [32,3 kB]
Holen:6 http://ports.ubuntu.com/ubuntu-ports mantic/restricted i386 Packages [32,3 kB]
Ign:6 http://ports.ubuntu.com/ubuntu-ports mantic/restricted i386 Packages
Ign:10 http://ports.ubuntu.com/ubuntu-ports mantic/universe i386 Packages
Ign:11 http://ports.ubuntu.com/ubuntu-ports mantic/multiverse i386 Packages
Ign:4 http://ports.ubuntu.com/ubuntu-ports mantic/main i386 Packages
Ign:6 http://ports.ubuntu.com/ubuntu-ports mantic/restricted i386 Packages
Ign:10 http://ports.ubuntu.com/ubuntu-ports mantic/universe i386 Packages
Ign:11 http://ports.ubuntu.com/ubuntu-ports mantic/multiverse i386 Packages
Ign:4 http://ports.ubuntu.com/ubuntu-ports mantic/main i386 Packages
Ign:6 http://ports.ubuntu.com/ubuntu-ports mantic/restricted i386 Packages
Ign:10 http://ports.ubuntu.com/ubuntu-ports mantic/universe i386 Packages
Ign:12 http://ports.ubuntu.com/ubuntu-ports mantic-updates/main i386 Packages
Ign:13 http://ports.ubuntu.com/ubuntu-ports mantic-updates/restricted i386 Packages
Ign:14 http://ports.ubuntu.com/ubuntu-ports mantic-updates/universe i386 Packages
Ign:15 http://ports.ubuntu.com/ubuntu-ports mantic-updates/multiverse i386 Packages
Ign:11 http://ports.ubuntu.com/ubuntu-ports mantic/multiverse i386 Packages
Ign:4 http://ports.ubuntu.com/ubuntu-ports mantic/main i386 Packages
Ign:6 http://ports.ubuntu.com/ubuntu-ports mantic/restricted i386 Packages
Ign:10 http://ports.ubuntu.com/ubuntu-ports mantic/universe i386 Packages
Ign:12 http://ports.ubuntu.com/ubuntu-ports mantic-updates/main i386 Packages
Ign:13 http://ports.ubuntu.com/ubuntu-ports mantic-updates/restricted i386 Packages
Ign:14 http://ports.ubuntu.com/ubuntu-ports mantic-updates/universe i386 Packages
Ign:15 http://ports.ubuntu.com/ubuntu-ports mantic-updates/multiverse i386 Packages
Ign:11 http://ports.ubuntu.com/ubuntu-ports mantic/multiverse i386 Packages
Ign:4 http://ports.ubuntu.com/ubuntu-ports mantic/main i386 Packages
Ign:16 http://ports.ubuntu.com/ubuntu-ports mantic-security/main i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports mantic-security/restricted i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports mantic-security/universe i386 Packages
Ign:19 http://ports.ubuntu.com/ubuntu-ports mantic-security/multiverse i386 Packages
Ign:6 http://ports.ubuntu.com/ubuntu-ports mantic/restricted i386 Packages
Ign:10 http://ports.ubuntu.com/ubuntu-ports mantic/universe i386 Packages
Ign:12 http://ports.ubuntu.com/ubuntu-ports mantic-updates/main i386 Packages
Ign:13 http://ports.ubuntu.com/ubuntu-ports mantic-updates/restricted i386 Packages
Ign:14 http://ports.ubuntu.com/ubuntu-ports mantic-updates/universe i386 Packages
Ign:15 http://ports.ubuntu.com/ubuntu-ports mantic-updates/multiverse i386 Packages
Ign:11 http://ports.ubuntu.com/ubuntu-ports mantic/multiverse i386 Packages
Fehl:4 http://ports.ubuntu.com/ubuntu-ports mantic/main i386 Packages
  404  Not Found [IP: 185.125.190.36 80]
Ign:16 http://ports.ubuntu.com/ubuntu-ports mantic-security/main i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports mantic-security/restricted i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports mantic-security/universe i386 Packages
Ign:19 http://ports.ubuntu.com/ubuntu-ports mantic-security/multiverse i386 Packages
Ign:6 http://ports.ubuntu.com/ubuntu-ports mantic/restricted i386 Packages
Ign:10 http://ports.ubuntu.com/ubuntu-ports mantic/universe i386 Packages
Ign:12 http://ports.ubuntu.com/ubuntu-ports mantic-updates/main i386 Packages
Ign:13 http://ports.ubuntu.com/ubuntu-ports mantic-updates/restricted i386 Packages
Ign:14 http://ports.ubuntu.com/ubuntu-ports mantic-updates/universe i386 Packages
Ign:15 http://ports.ubuntu.com/ubuntu-ports mantic-updates/multiverse i386 Packages
Ign:16 http://ports.ubuntu.com/ubuntu-ports mantic-security/main i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports mantic-security/restricted i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports mantic-security/universe i386 Packages
Ign:19 http://ports.ubuntu.com/ubuntu-ports mantic-security/multiverse i386 Packages
Ign:12 http://ports.ubuntu.com/ubuntu-ports mantic-updates/main i386 Packages
Ign:13 http://ports.ubuntu.com/ubuntu-ports mantic-updates/restricted i386 Packages
Ign:14 http://ports.ubuntu.com/ubuntu-ports mantic-updates/universe i386 Packages
Ign:15 http://ports.ubuntu.com/ubuntu-ports mantic-updates/multiverse i386 Packages
Ign:16 http://ports.ubuntu.com/ubuntu-ports mantic-security/main i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports mantic-security/restricted i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports mantic-security/universe i386 Packages
Ign:19 http://ports.ubuntu.com/ubuntu-ports mantic-security/multiverse i386 Packages
Fehl:12 http://ports.ubuntu.com/ubuntu-ports mantic-updates/main i386 Packages
  404  Not Found [IP: 185.125.190.36 80]
Ign:13 http://ports.ubuntu.com/ubuntu-ports mantic-updates/restricted i386 Packages
Ign:14 http://ports.ubuntu.com/ubuntu-ports mantic-updates/universe i386 Packages
Ign:15 http://ports.ubuntu.com/ubuntu-ports mantic-updates/multiverse i386 Packages
Ign:16 http://ports.ubuntu.com/ubuntu-ports mantic-security/main i386 Packages
Ign:17 http://ports.ubuntu.com/ubuntu-ports mantic-security/restricted i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports mantic-security/universe i386 Packages
Ign:19 http://ports.ubuntu.com/ubuntu-ports mantic-security/multiverse i386 Packages
Fehl:16 http://ports.ubuntu.com/ubuntu-ports mantic-security/main i386 Packages
  404  Not Found [IP: 185.125.190.36 80]
Ign:17 http://ports.ubuntu.com/ubuntu-ports mantic-security/restricted i386 Packages
Ign:18 http://ports.ubuntu.com/ubuntu-ports mantic-security/universe i386 Packages
Ign:19 http://ports.ubuntu.com/ubuntu-ports mantic-security/multiverse i386 Packages
Paketlisten werden gelesen… Fertig
E: Fehlschlag beim Holen von http://ports.ubuntu.com/ubuntu-ports/dists/mantic/main/binary-i386/Packages 404  Not Found [IP: 185.125.190.36 80]
E: Fehlschlag beim Holen von http://ports.ubuntu.com/ubuntu-ports/dists/mantic-updates/main/binary-i386/Packages 404  Not Found [IP: 185.125.190.36 80]
E: Fehlschlag beim Holen von http://ports.ubuntu.com/ubuntu-ports/dists/mantic-security/main/binary-i386/Packages 404  Not Found [IP: 185.125.190.36 80]
E: Einige Indexdateien konnten nicht heruntergeladen werden. Sie wurden ignoriert oder alte an ihrer Stelle benutzt.

**Reply by Conan179** [reply]
Dein Problem liegt an deiner /etc/apt/sources.list die solltest du mal überprüfen.

**Reply by FireDVB** [reply]
da steht das drin:

deb http://ports.ubuntu.com/ubuntu-ports mantic main restricted universe multiverse
deb http://ports.ubuntu.com/ubuntu-ports/ mantic-updates main restricted universe multiverse
deb http://ports.ubuntu.com/ubuntu-ports/ mantic-security main restricted universe multiverse

**Reply by FireDVB** [reply]
Habe die sources.list geupdated von einem anderen ubuntu64 23.10 und trotzdem werden die Sachen nicht installiert oder aktualisiert.

 ./prerequisites check Ubuntu23

ERROR: The program 'libtool' was not found in path
ERROR: The program 'svn' was not found in path
ERROR: The header file 'sys/acl.h' was not found in /usr/(local/)include
ERROR: The header file 'sys/capability.h' was not found in /usr/(local/)include
ERROR: The header file 'uuid/uuid.h' was not found in /usr/(local/)include
ERROR: The header file 'openssl/evp.h' was not found in /usr/(local/)include
ERROR: The header file 'gnutls/gnutls.h' was not found in /usr/(local/)include
ERROR: The header file 'sqlite3.h' was not found in /usr/(local/)include
ERROR: The pkgconf 'libzstd.pc' was not found in /usr/(local/)lib*/(*/)pkgconfig/


sudo apt install libtool
Paketlisten werden gelesen… Fertig
Abhängigkeitsbaum wird aufgebaut… Fertig
Statusinformationen werden eingelesen… Fertig
libtool ist schon die neueste Version (2.4.7-7).
0 aktualisiert, 0 neu installiert, 0 zu entfernen und 0 nicht aktualisiert.

**Comment by Prisrak1** [comment]
> Dein Problem liegt an deiner /etc/apt/sources.list die solltest du mal überprüfen.

Scheinbar läuft es bei dir auf irgendeiner Raspberry Pi, stelle uns doch bitte deinem Backup von deinem Image, wo es auch tatsächlich funktioniert.

**Reply by Moppelchen** [reply]
Hat es denn jmd. verbrieft geschafft, sämtliche prerequisites zu erfüllen und lauffähig zu bekommen auf einem RPi4/5? Ich verplempere viel Zeit beim erfolglosen recherchieren, wo und wie die fehlenden Pakete herzuzaubern sind? Der ganze i386-Stuff wird stets ignoriert und z.B. ncurses-dev sind wohl virtuelle Developer-Pakete, die nicht wirklich greifbar scheinen? Falls jmd. eine funktionierende sources.list zur Hand hätte, wäre mir und sicherlich vielen anderen hier sehr geholfen, den Frust abzubauen.

**Reply by fda77** [reply]
https://freetz-ng.github.io/

**Reply by Moppelchen** [reply]
Danke den Link habe ich schon zig-mal versucht. Nach sicherlich mehreren hundert Stunden Lesens und Probierens seit letzten Weihnachten scheitert  es halt immer an den fehlenden Quellen, die ich auch per Suche nicht wirklich finden kann. 
Ein übliches Bsp.
`rebecca@micha-desktop-pi4:~ $ umask 0022
rebecca@micha-desktop-pi4:~ $ git clone https://github.com/Freetz-NG/freetz-ng ~ /freetz-ng
Cloning into '/home/rebecca/freetz-ng'...
remote: Enumerating objects: 227390, done.
remote: Counting objects: 100% (4738/4738), done.
remote: Compressing objects: 100% (1482/1482), done.
remote: Total 227390 (delta 3270), reused 4625 (delta 3193), pack-reused 222652
Receiving objects: 100% (227390/227390), 79.52 MiB | 2.22 MiB/s, done.
Resolving deltas: 100% (158079/158079), done.
Updating files: 100% (9425/9425), done.
rebecca@micha-desktop-pi4:~ $ ./prerequisites install -y Ubuntu23
-bash: ./prerequisites: No such file or directory
rebecca@micha-desktop-pi4:~ $ cd freetz-ng
rebecca@micha-desktop-pi4:~ /freetz-ng$ cd tools
rebecca@micha-desktop-pi4:~ /freetz-ng/tools$ ./prerequisites install -y Ubuntu23

Packages for 'Ubuntu23':
autopoint bc binutils bison bsdmainutils bzip2 ccache cmake cpio curl ecj flex ftp g++ gawk gcc gcc-multilib gettext git graphicsmagick imagemagick inkscape intltool java-wrappers kmod lib32ncurses-dev lib32stdc++6 lib32z1-dev libacl1-dev libc6-dev-i386 libcap-dev libelf-dev libglib2.0-dev libgnutls28-dev libncurses-dev libreadline-dev libsqlite3-dev libssl-dev libstring-crc32-perl libtool-bin libusb-dev libxml2-dev libzstd-dev make ncftp netcat-openbsd net-tools patch patchutils perl pkg-config pv rsync sharutils sqlite3 subversion sudo texinfo tofrodos unar unzip uuid-dev wget zip zlib1g-dev

Ign:1 https://download.mono-project.com/repo/ubuntu stable-focal InRelease                       
OK:2 http://ports.ubuntu.com/ubuntu-ports lunar InRelease                                        
Ign:3 https://ppa.launchpadcontent.net/pinta-maintainers/pinta-stable/ubuntu lunar InRelease     
OK:4 http://ports.ubuntu.com/ubuntu-ports lunar-backports InRelease
Ign:5 https://ppa.launchpadcontent.net/shutter/ppa/ubuntu lunar InRelease
Holen:6 http://ports.ubuntu.com/ubuntu-ports lunar-updates InRelease [109 kB]
Fehl:7 https://ppa.launchpadcontent.net/pinta-maintainers/pinta-stable/ubuntu lunar Release
  404  Not Found [IP: 185.125.190.80 443]
Fehl:8 https://ppa.launchpadcontent.net/shutter/ppa/ubuntu lunar Release
  404  Not Found [IP: 185.125.190.80 443]
Holen:9 http://ports.ubuntu.com/ubuntu-ports lunar-security InRelease [109 kB]
Holen:10 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main arm64 Packages [401 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Holen:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages [255 kB]
Ign:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages                 
Ign:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages           
Ign:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages
Ign:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages
Ign:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages
Ign:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages               
Ign:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages                     
Ign:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages           
Ign:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages                     
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]    
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Holen:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages [149 kB]
Ign:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages
Ign:36 http://ports.ubuntu.com/ubuntu-ports lunar-security/main i386 Packages
Ign:37 http://ports.ubuntu.com/ubuntu-ports lunar-security/restricted i386 Packages          
Fehl:11 http://ports.ubuntu.com/ubuntu-ports lunar-updates/main i386 Packages                
  404  Not Found [IP: 185.125.190.36 80]
Ign:23 http://ports.ubuntu.com/ubuntu-ports lunar-updates/restricted i386 Packages           
Ign:36 http://ports.ubuntu.com/ubuntu-ports lunar-security/main i386 Packages
Ign:37 http://ports.ubuntu.com/ubuntu-ports lunar-security/restricted i386 Packages              
Ign:36 http://ports.ubuntu.com/ubuntu-ports lunar-security/main i386 Packages                    
Ign:37 http://ports.ubuntu.com/ubuntu-ports lunar-security/restricted i386 Packages              
Ign:36 http://ports.ubuntu.com/ubuntu-ports lunar-security/main i386 Packages
Ign:37 http://ports.ubuntu.com/ubuntu-ports lunar-security/restricted i386 Packages
Ign:36 http://ports.ubuntu.com/ubuntu-ports lunar-security/main i386 Packages
Ign:37 http://ports.ubuntu.com/ubuntu-ports lunar-security/restricted i386 Packages              
Fehl:36 http://ports.ubuntu.com/ubuntu-ports lunar-security/main i386 Packages                   
  404  Not Found [IP: 185.125.190.36 80]
Ign:37 http://ports.ubuntu.com/ubuntu-ports lunar-security/restricted i386 Packages              
Ign:1 https://download.mono-project.com/repo/ubuntu stable-focal InRelease                       
OK:1 https://download.mono-project.com/repo/ubuntu stable-focal InRelease
Paketlisten werden gelesen… Fertig                          
E: Das Depot »https://ppa.launchpadcontent.net/pinta-maintainers/pinta-stable/ubuntu lunar Release« enthält keine Release-Datei.
N: Eine Aktualisierung von solch einem Depot kann nicht auf eine sichere Art durchgeführt werden, daher ist es standardmäßig deaktiviert.
N: Weitere Details zur Erzeugung von Paketdepots sowie zu deren Benutzerkonfiguration finden Sie in der Handbuchseite apt-secure(8).
E: Das Depot »https://ppa.launchpadcontent.net/shutter/ppa/ubuntu lunar Release« enthält keine Release-Datei.
N: Eine Aktualisierung von solch einem Depot kann nicht auf eine sichere Art durchgeführt werden, daher ist es standardmäßig deaktiviert.
N: Weitere Details zur Erzeugung von Paketdepots sowie zu deren Benutzerkonfiguration finden Sie in der Handbuchseite apt-secure(8).
N: Das Laden der konfigurierten Datei »main/binary-i386/Packages« wird übersprungen, da das Depot »https://download.mono-project.com/repo/ubuntu stable-focal InRelease« die Architektur »i386« nicht unterstützt.
rebecca@micha-desktop-pi4:~/freetz-ng/tools$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 23.04
Release:	23.04
Codename:	lunar`


Deshalb meine höfliche Bitte/Frage, wie die sources.list die notwendigen und richtigen Quellen findet? Ansonsten kann ich mich bis zum St.-Nimmerleinstag im Kreise drehen.
Many TX
